Classes/BITCrashManager.m

```diff
@@ -556,6 +556,7 @@ - (void) registerObservers {
             self.didLogLowMemoryWarning = YES;
           }
         });
+        dispatch_resume(source);
```

Comment:
I'm not sure why we need that here. Can you explain?

Reply:
We need this because dispatch_source_create creates source in a suspended state and we must call at some point after creation dispatch_resume. https://developer.apple.com/documentation/dispatch/1385630-dispatch_source_create?language=objc